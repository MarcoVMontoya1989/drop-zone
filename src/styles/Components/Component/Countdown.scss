.clock-container {
  display: flex;
  justify-content: center;
  flex-direction: row;
  align-items: center;
  height: 10rem;
  background-color: transparent;
}

.clock-column {
  margin-right: 2rem;
  text-align: center;
  position: relative;

  &:last-child {
    &::after {
      display: none;
    }
  }
}

.clock-column:last-child {
  margin-right: 0;
}

.clock-timer {
  display: flex;
  justify-content: center;
  align-items: center;

  width: 58px;
  height: 73px;

  background-color: black;
  color: #FFF;
  font-family: $rozha-normal;
  font-style: normal;
  font-weight: normal;
  font-size: 48px;
  line-height: 40px;

  @media screen and (max-width: 768px) {
    width: 31.38px;
    height: 39.5px;
    font-size: 24px;
  }
}

.clock-label {
  padding-top: 25px;
  text-transform: uppercase;
  color: black;
  text-align: center;
  font-family: $roboto-normal;
  font-size: 24px;
  font-style: normal;
  font-weight: 900;
  line-height: 28px;
  letter-spacing: 0.15em;

  @media screen and (max-width: 768px) {
    font-size: 14px;
    line-height: 16px;
  }
}
