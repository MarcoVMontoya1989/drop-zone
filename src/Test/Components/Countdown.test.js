import {shallow} from "enzyme";
import {render, screen} from '@testing-library/react';

import CountdownComponent from "../../Components/Component/Countdown.component";
import Countdown from "react-countdown";

const wrapper = shallow(<CountdownComponent />);

describe('Countdown Tests', () => {
  it('should render component', () => {
    const CountdownRenderer = render(<CountdownComponent />);
    expect(CountdownRenderer).toMatchSnapshot();
  });

  it('should render Countdown timer library in the component', () => {
    expect(wrapper.containsMatchingElement(<Countdown />)).toEqual(true);
    
  });
});