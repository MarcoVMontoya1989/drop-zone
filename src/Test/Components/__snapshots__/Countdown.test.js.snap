// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Countdown Tests should render component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="container clock-container"
      >
        <div
          class="clock-column"
        >
          <div
            class="clock-count"
          >
            <p
              class="clock-day clock-timer"
            >
              1
            </p>
            <p
              class="clock-day clock-timer"
            >
              3
            </p>
          </div>
          <p
            class="clock-label"
          >
            Days
          </p>
        </div>
        <div
          class="clock-column"
        >
          <div
            class="clock-count"
          >
            <p
              class="clock-hours clock-timer"
            >
              0
            </p>
            <p
              class="clock-hours clock-timer"
            >
              8
            </p>
          </div>
          <p
            class="clock-label"
          >
            Hours
          </p>
        </div>
        <div
          class="clock-column"
        >
          <div
            class="clock-count"
          >
            <p
              class="clock-minutes clock-timer"
            >
              1
            </p>
            <p
              class="clock-minutes clock-timer"
            >
              2
            </p>
          </div>
          <p
            class="clock-label"
          >
            Mins
          </p>
        </div>
        <div
          class="clock-column"
        >
          <div
            class="clock-count"
          >
            <p
              class="clock-seconds clock-timer"
            >
              0
            </p>
            <p
              class="clock-seconds clock-timer"
            >
              5
            </p>
          </div>
          <p
            class="clock-label"
          >
            Secs
          </p>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="container clock-container"
    >
      <div
        class="clock-column"
      >
        <div
          class="clock-count"
        >
          <p
            class="clock-day clock-timer"
          >
            1
          </p>
          <p
            class="clock-day clock-timer"
          >
            3
          </p>
        </div>
        <p
          class="clock-label"
        >
          Days
        </p>
      </div>
      <div
        class="clock-column"
      >
        <div
          class="clock-count"
        >
          <p
            class="clock-hours clock-timer"
          >
            0
          </p>
          <p
            class="clock-hours clock-timer"
          >
            8
          </p>
        </div>
        <p
          class="clock-label"
        >
          Hours
        </p>
      </div>
      <div
        class="clock-column"
      >
        <div
          class="clock-count"
        >
          <p
            class="clock-minutes clock-timer"
          >
            1
          </p>
          <p
            class="clock-minutes clock-timer"
          >
            2
          </p>
        </div>
        <p
          class="clock-label"
        >
          Mins
        </p>
      </div>
      <div
        class="clock-column"
      >
        <div
          class="clock-count"
        >
          <p
            class="clock-seconds clock-timer"
          >
            0
          </p>
          <p
            class="clock-seconds clock-timer"
          >
            5
          </p>
        </div>
        <p
          class="clock-label"
        >
          Secs
        </p>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
